test/lsregion.c

```diff
@@ -7,8 +7,12 @@
 
 /* For the sake of lsregion_alloc_object (it demands alignof()) */
 #ifndef offsetof
-#define offsetof(type, member) ((size_t) &((type *)0)->member)
-#endif
+#  if defined(__has_builtin) && __has_builtin(__builtin_offsetof)
+#    define offsetof(type, member) __builtin_offsetof(type, member)
+#  else
+#    define offsetof(type, member) ((size_t)&((type *)0)->member)
+#  endif
```

Comment:
Let's move it to `include/small/util.h` to avoid code duplication.

Reply:
Done

Reply:
Done